test/services/permissions.test.js

```diff
@@ -1,9 +1,9 @@
 const nock = require('nock');
 const PermissionsChecker = require('../../src/services/permissions-checker');
-const ServiceUrlGetter = require('../../src/services/service-url-getter');
+const forestUrlGetter = require('../../src/utils/forest-url-getter');
```

Comment:
[THINKING] You might want to use the inject() methods here, knowing that the file might be deleted